package.json

```diff
@@ -49,7 +50,7 @@
   "scripts": {
     "format": "standard --fix",
     "test-api": "tape test | tap-spec",
-    "test-coverage": "nyc --reporter lcov tape test/index.js | tap-spec",
+    "test-coverage": "c8 --check-coverage --100 --reporter lcov npm run test-api",
```

Comment:
`c8` uses native v8 coverage gathering so it’s faster and closer to how Node works, and it’s better at ESM